packages/database/src/TupaiaDatabase.js

```diff
@@ -685,15 +695,31 @@ function addJoin(baseQuery, recordType, joinOptions) {
 
 function getColSelector(connection, inputColStr) {
```

Comment:
This implementation assumes we never have a column selector which **both** calls `COALESCE()` **and** uses JSON selectors

Is this a concern?